modules/sor/sorV2/lib/pools/gyro2/gyro2Pool.ts

```diff
@@ -229,4 +229,38 @@ export class Gyro2Pool implements BasePool {
 
         return { tIn, tOut, sqrtAlpha, sqrtBeta };
     }
+
+    addLiquiditySingleTokenExactIn(tokenIn: Token, bpt: Token, amount: TokenAmount): TokenAmount {
```

Comment:
Ok, so following up on what we discussed at the on-site, we'll stick with only Weighted/Stable pools on this first iteration, right?
We'll still go with previous pool math structure for this PR, so I think you'll need to keep these placeholder methods as an "intermediary state" for now.
The idea is to setup the V3 math structure on a following PR and use it for all V3 operations.
I'm not sure how the implementation will end up looking like, but I'd assume that at that point you'll be able to remove these placeholder methods to cleanup the code related to V2 math.
Later on, we should be able to evaluate if it's ok to move everything to the new math structure as ideally it should yield the same result.